packages/mcp/package.json

```diff
@@ -82,8 +82,7 @@
   "wireit": {
     "build": {
       "dependencies": [
-        "compile",
-        "lint"
+        "compile"
```

Comment:
Eric and I discovered that you're going to need to modify the `.sfdevrc` file to prevent it from just putting the `lint` back in.
```
  "wireit": {
    "build": {
      "dependencies": [
        "compile"
      ]
    },
    "test": {
      "dependencies": [
        "test:only",
        "test:compile",
        "link-check"
      ]
    }
  }
```